package.json

```
@@ -3,6 +3,13 @@
   "version": "0.11.2",
   "main": "build/index.js",
   "types": "build/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./build/index.d.ts",
+      "default": "./build/index.js"
+    },
+    "./package.json": "./package.json"
```

Should `bin` entry point be included as well? Or? https://github.com/jest-community/create-jest-runner/blob/949fa9fd5ad35caca4853ca679416c699f7a9489/package.json#L73

```suggestion
    "./package.json": "./package.json",
    "./generator/index.js": "./generator/index.js"
```

---

not sure how `bin` works... I don't think it's needed, but doesn't hurt!